deepchem/feat/smiles_tokenizer.py

```diff
@@ -0,0 +1,292 @@
+# Requriments - transformers, tokenizers
+# Right now, the Smiles Tokenizer uses an exiesting vocab file from rxnfp that is fairly comprehensive and from the USPTO dataset.
+# The vocab may be expanded in the near future
+
+import collections
+import logging
+import os
+import re
+import numpy as np
+import pkg_resources
+from typing import List
+from transformers import BertTokenizer
```

Comment:
Can you expose `SmilesTokenizer` and `BasicSmilesTokenizer` in the top level `deepchem/feat/__init__.py` with an import guard?

Reply:
for this, would the following blurb work?

```
try:
    import transformers
    from transformers import BertTokenizer

    from deepchem.feat.smiles_tokenizer import SmilesTokenizer
    from deepchem.feat.smiles_tokenizer import BasicSmilesTokenizer
except ImportError:
    logger.warning("HuggingFace transformers is not available. Please install using 'pip install transformers' to use the SmilesTokenizer")
```

Reply:
added this for now, feel free to let me know if any changes necessary

Reply:
See my comment above (basically, we should run `yapf` and use `ModuleNotFoundError` instead of `ImportError`